src/box/txn.h

```diff
@@ -119,6 +119,8 @@ enum txn_flag {
 	TXN_IS_STARTED_IN_ENGINE = 0x400,
 	/** Transaction supports multiversion concurrency control. */
 	TXN_SUPPORTS_MVCC = 0x800,
+	/** Set during statement rollback. */
```

Comment:
Please improve the comment - say how and why it's set.

Reply:
Done.

```diff
-       /** Set during statement rollback. */
+       /**
+        * Set during statement rollback.
+        *
+        * If statetement is rolled back we should not fail due to OOM.
+        * This flag is set when statement rollback is started. If it is set
+        * then memory allocated even if memory limit is reached.
+        */
```